include/openmc/mgxs_interface.h

```diff
@@ -46,6 +46,9 @@ class MgxsInterface {
   // Get the kT values which are used in the OpenMC model
   vector<vector<double>> get_mat_kTs();
 
+  // Get the group index corresponding to a continuous energy
+  int get_group_index(double e);
```

Comment:
```suggestion
  int get_group_index(double E);
```


https://docs.openmc.org/en/latest/devguide/styleguide.html#naming